_posts/2018-01-01-futures-0.3.0-alpha.3.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Futures 0.3.0-alpha.3"
+author: "Josef Brandl"
+author_github: "MajorBreakfast"
+date:   2018-01-01
+categories: blog
+---
+
+This release introduces the `spawn!` and `spawn_with_handle!` macros for use in async functions, closures and blocks. Both macros let you spawn a task that polls the given future to completion. The task is spawned onto the current context's executor.
+
+`spawn!` lets you spawn futures with `Output = ()`:
+
+```rust
+#![feature(async_await, await_macro, futures_api)]
+#[macro_use] extern crate futures;
+
+spawn!(future).unwrap();
+```
+
+`spawn_with_handle!` lets you spawn futures with any `Output` type and also lets you await the output:
```

Comment:
Maybe mention the `Output: Send` requirement?

Reply:
Oh, just noticed that this is below, not sure if it needs repeating here then, but I missed it during my first reading.

Reply:
Good idea. Mentioning it twice doesn't hurt.

Ich changed it to:
`spawn_with_handle!` lets you spawn futures with any `Output: Send` type and also lets you await the produced output value